docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -4,228 +4,302 @@ description: How to configure Teleport access with Google Workspace (formerly kn
 videoBanner: WTLWc6nnPfk
 ---
 
-This guide will explain how to configure [Google Workspace](https://workspace.google.com/)
-to be a single sign-on (SSO) provider to issue Teleport credentials to specific
-groups of users. When used in combination with role-based access control (RBAC)
-it allows Teleport administrators to define policies like:
+This guide explains how to configure [Google Workspace](https://workspace.google.com/)
+to be a single sign-on (SSO) provider that issues Teleport credentials to specific
+groups of users. When you use Google Workspace in combination with Teleport role-based 
+access control (RBAC), you can define policies like the following:
 
 - Only members of the "DBA" Google group can connect to PostgreSQL databases.
-- Developers must never SSH into production servers.
+- Developers must never use SSH to access production servers.
 
 ## Prerequisites
 
-Before you get started you’ll need:
-
-- A Google Workspace super administrator account. We recommend setting up a
-  separate super admin account with 2FA as opposed to granting your daily user
-  super admin privileges.
-- Ability to create a Google Cloud project, which requires signing up for Google
-  Cloud. Note that this guide will not require using any paid Google Cloud services.
-- Ability to set up Google Workspace groups.
-- Teleport role with access to maintaining `oidc` resources.  This is available
-  in the default `editor` role.
-
-(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+Before you get started, verify the following:
+
+- You have a Google Workspace super administrator account. 
+   As a best practice, you should set up a separate account that requires multifactor 
+   authentication to perform administrative actions. In most cases, you should avoid 
+   granting your own sign-in user account elevated administrative privileges.
+- You can sign up for Google Cloud and create a Google Cloud project. 
+   Note that this guide doesn't require using any paid Google Cloud services.
+- You can set up Google Workspace groups.
+- You have a Teleport role with permission to maintain `oidc` resources. 
+   This permission is available in the default `editor` role.
```

Comment:
```suggestion
   This permission is available in the preset `editor` role.
```

These roles are called "presets" so the more consistently we use that language, the easier we make it for users to find https://goteleport.com/docs/access-controls/reference/#preset-roles